recipes/newrelic/infrastructure/ohi/kafka/windows.yml

```diff
@@ -0,0 +1,61 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: kafka-open-source-integration
+displayName: kafka Open Source Integration
+description: New Relic install recipe for default kafka Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-kafka
+
+dependencies:
+  - infrastructure-agent-installer
+
+installTargets:
+  - type: host
+    os: windows
+    kernelArch: x86_64
+
+keywords:
+  - Infrastructure
+  - Integration
+  - kafka
+
+# Kafka runs under java process in windows, so we checking if kafka port 9092 is running under Java
+processMatch:
+  - java
+
+preInstall:
+  requireAtDiscovery: |
+    powershell -command '
+    try{
+      $process = Get-Process -Id (Get-NetTCPConnection -LocalPort 9092).OwningProcess | select name -ErrorAction SilentlyContinue
```

Comment:
@NRhzhao unlike linux which has a process detect specific to kafka, for windows I don't think we should rely on java with port 9092 to determine kafka is used, this can be any other java app that happens to run on that port.
Is there something else we could latch on to ensure kafka is running?

Reply:
@Julien4218 Let me digging into this a little more.